.goreleaser.yml

```diff
@@ -44,14 +45,12 @@ nfpm:
   name_template: '{{ .ProjectName }}_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}'
   homepage:  https://github.com/leopardslab/Dunner
   description: A Docker based task runner tool
+  maintainer: Milindu Sanoj Kumarage <agentmilindu@gmail.com>
```

Comment:
Let's change this to LeaopardsLabs community

Reply:
Do we have an email id for that? @agentmilindu